packages/marginfi-client-v2/src/client.ts

```diff
@@ -1098,7 +1137,7 @@ class MarginfiClient {
           return signature;
         }
 
-        await sleep(500); // Wait before retrying
+        await sleep(1000); // Wait before retrying
```

Comment:
Why did this change?

Reply:
didn't mean to change this, good you pointed this out